run/cifs_archive/verify-and-configure-archive.sh

```diff
@@ -120,23 +120,19 @@ function configure_archive () {
   local credentials_file_path="/root/.teslaCamArchiveCredentials"
   /tmp/write-archive-configs-to.sh "$credentials_file_path"
 
-  if ! grep -w -q "$archive_path" /etc/fstab
-  then
-    local sharenameforstab="${SHARE_NAME// /\\040}"
-    echo "//$ARCHIVE_SERVER/$sharenameforstab $archive_path cifs noauto,credentials=${credentials_file_path},iocharset=utf8,file_mode=0777,dir_mode=0777,$VERS_OPT,$SEC_OPT 0" >> /etc/fstab
-  fi
+  sed -i "/$archive_path/ d" /etc/fstab
```

Comment:
It's much better to generically identify the Tesla-USB entries, then it doesn't matter what it was set to before. Luckily it seems this is easy because the keyword ".teslaCamArchiveCredentials" is hard-coded into the CIFS options. Hence the following RegEx should work every time:

`^.*\.teslaCamArchiveCredentials.*$`

I tested it using an online tool and my actual FsTab file and it appears to work:

![image](https://user-images.githubusercontent.com/1496225/104945327-db1e9080-59b8-11eb-8121-156509b7198c.png)

Reply:
Ah, I was focused on trying to identify the mount points (which also don't change), and that requires escaping them properly.  Thanks for the hint to look at something that doesn't require escaping!